acl_loader/main.py

```diff
@@ -796,12 +796,16 @@ def show_table(self, table_name):
             stage = val.get("stage", Stage.INGRESS).lower()
 
             if val["type"] == AclLoader.ACL_TABLE_TYPE_CTRLPLANE:
-                services = natsorted(val["services"])
-                data.append([key, val["type"], services[0], val["policy_desc"], stage])
+                services = val.get("services", [])
```

Comment:
>services [](http://example.com/codeflow?start=16&length=8)

Do not reuse var name, suggest `service_array`

Reply:
I've fixed it